### PR TITLE
Performance

### PR DESCRIPTION
**Pull request by mathieupost**
Much performance, such fast!

### CODE REVIEW

File: src/main/java/tudelft/ti2806/pl3/data/graph/GraphDataRepository.java [hunk 1]
```diff
@@ -153,9 +153,7 @@ protected static void addRefLabels(DataNode node, GeneData geneData) {
 		boolean started = false;
 		for (int i = start; i <= end; i++) {
 			if (started) {
-				if (g != null) {
```
Comment (TomBrouws) [inline src/main/java/tudelft/ti2806/pl3/data/graph/GraphDataRepository.java]: Why have you changed my code? 😥

**Comment by TomBrouws** [comment]
Looks good, but not sure if removal of my null check can be done without consequences. Need to check

**Comment by mathieupost** [comment]
Intellij said it would never be null if started is true. And if you look at the code, that makes sense.

File: src/main/java/tudelft/ti2806/pl3/data/graph/GraphDataRepository.java (hunk 1 above)
Comment (mathieupost) [inline src/main/java/tudelft/ti2806/pl3/data/graph/GraphDataRepository.java]: Well because g is never null, G.

**Comment by Delfil** [comment]
All fine

**Comment by TomBrouws** [comment]
Ah sure Mathieu, that makes sense. Didn't see that when writing it.

**Pull request merged**